packages/common/components/MediaViewer/sub-components/PlayerVolumeControl/PlayerVolumeControl.styled.ts

```diff
@@ -0,0 +1,147 @@
+import { tablet } from "@docspace/components/utils/device";
+import { isMobile } from "react-device-detect";
+import styled, { css } from "styled-components";
+
+export const PlayerVolumeControlWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: 10px;
+`;
+
+export const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+
+const mobilecss = css`
```

Comment:
Use comma with the same body

input[type="range"]::-webkit-slider-thumb,
input[type="range"]::-moz-range-thumb,
input[type="range"]::-ms-fill-upper {
 ...
}